releasenotes/notes/nodeps_intoto-208171c6e23123a0.yaml

```diff
@@ -0,0 +1,15 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Add ``--in-toto`` flag to ``datadog-agent integration`` command to enable in-toto
+  - Add ``--tuf-log`` flag to ``datadog-agent integration`` command to enable TUF logging
+enhancements:
+  - |
+    Install integration wheels with ``--no-deps`` to prevent installation of unwanted dependencies
```

Comment:
nit: more context on this release note would be appreciated :) (most users probably don't know what `integration wheels` are at this point, so let's mention the `datadog-agent integration` command again). Also, `--no-deps` is an implementation detail I'd say, maybe we could simply mention that the command does not pull any of the integration's dependencies.

^ all this is just to ensure that someone with no context on the wheel project understands whether they should care about this at all